packages/components/package.json

```diff
@@ -1,83 +1,83 @@
 {
-    "name": "@deriv/components",
-    "version": "1.0.0",
-    "description": "deriv components",
-    "main": "src/index.js",
-    "private": true,
-    "scripts": {
-        "start": "rimraf lib && npm run test && npm run serve",
-        "serve": "rimraf lib && cross-env BUILD_MODE='serve' webpack --mode=production --progress --watch",
-        "build": "rimraf lib && npm run build:icons && webpack --mode=production --progress && npm run build:icons-manifest",
```

Comment:
I think `npm run build:icons-manifest` was removed by mistake. Can you undo it?

Reply:
done.